packages/forms/frontend/sirius-components-forms/src/propertysections/SplitButtonPropertySection.tsx

```diff
@@ -0,0 +1,322 @@
+    const props: ButtonStyleProps = {
+      backgroundColor: action.style?.backgroundColor ?? null,
+      foregroundColor: action.style?.foregroundColor ?? null,
+      fontSize: action.style?.fontSize ?? null,
+      italic: action.style?.italic ?? null,
+      bold: action.style?.bold ?? null,
+      underline: action.style?.underline ?? null,
+      strikeThrough: action.style?.strikeThrough ?? null,
+      iconOnly: action.buttonLabel ? false : true,
+    };
+    return useStyle(props);
+  });
+
+  const [pushButton, { loading, data, error }] = useMutation<GQLPushButtonMutationData, GQLPushButtonMutationVariables>(
+    pushButtonMutation
+  );
+
+  useEffect(() => {
+    if (!loading) {
+      if (error) {
+        addErrorMessage('An unexpected error has occurred, please refresh the page');
+      }
+      if (data) {
+        const { pushButton } = data;
+        if (isErrorPayload(pushButton) || isSuccessPayload(pushButton)) {
+          addMessages(pushButton.messages);
+        }
+      }
+    }
+  }, [loading, error, data]);
+
+  const [
+    updateWidgetFocus,
+    { loading: updateWidgetFocusLoading, data: updateWidgetFocusData, error: updateWidgetFocusError },
+  ] = useMutation<GQLUpdateWidgetFocusMutationData, GQLUpdateWidgetFocusMutationVariables>(updateWidgetFocusMutation);
+
+  const sendUpdateWidgetFocus = (selected: boolean) => {
+    const input: GQLUpdateWidgetFocusInput = {
+      id: crypto.randomUUID(),
+      editingContextId,
+      representationId: formId,
+      widgetId: widget.id,
+      selected,
+    };
+    const variables: GQLUpdateWidgetFocusMutationVariables = {
+      input,
+    };
+    updateWidgetFocus({ variables });
+  };
+
+  useEffect(() => {
+    if (!updateWidgetFocusLoading) {
+      if (updateWidgetFocusError) {
+        addErrorMessage('An unexpected error has occurred, please refresh the page');
+      }
+      if (updateWidgetFocusData) {
+        const { updateWidgetFocus } = updateWidgetFocusData;
+        if (isErrorPayload(updateWidgetFocus)) {
+          addMessages(updateWidgetFocus.messages);
+        }
+      }
+    }
+  }, [updateWidgetFocusLoading, updateWidgetFocusData, updateWidgetFocusError]);
+
+  const onFocus = () => sendUpdateWidgetFocus(true);
+  const onBlur = () => {
+    sendUpdateWidgetFocus(false);
+  };
+
+  const handleClick = () => {
+    const button: GQLButton = widget.actions[state.selectedIndex];
+    const input: GQLPushButtonInput = {
+      id: crypto.randomUUID(),
+      editingContextId,
+      representationId: formId,
+      buttonId: button.id,
+    };
+    const variables: GQLPushButtonMutationVariables = { input };
+    pushButton({ variables });
+  };
+
+  const handleMenuItemClick = (_event, index) => {
+    setState((prevState) => ({ ...prevState, open: false, selectedIndex: index }));
+  };
+
+  const handleToggle = () => {
+    setState((prevState) => ({ ...prevState, open: !prevState.open }));
+  };
+
+  const handleClose = (_event) => {
```

Comment:
There is something wrong here, you put and underscore in front of `event` parameter but you use it inside the method.